benchmark/scripts/Benchmark_Driver

```diff
@@ -55,6 +55,9 @@ class BenchmarkDriver(object):
         self.tests = tests or self._get_tests()
         self.parser = parser or LogParser()
         self.results = {}
+        # Set a constant hash seed. Some tests are currently sensitive to
+        # fluctuations in the number of hash collisions.
+        os.environ['SWIFT_DETERMINISTIC_HASHING'] = '1'
```

Comment:
That's an important change. It should be in a separate commit or at least be mentioned in the commit message

Reply:
I’m not sure what you mean. Deterministic hashing env variable was [added some 5 months ago by @lorentey](https://github.com/apple/swift/blame/master/benchmark/scripts/Benchmark_Driver#L420). I’ve just added a unit test for it and moved the implementation into `BenchmarkDriver` class.